docs/resources/serving_endpoint.md

```diff
@@ -0,0 +1,88 @@
+---
+subcategory: "MLflow"
+---
+# databricks_serving_endpoint Resource
+
+This resource allows you to manage [Model Serving Endpoints](https://docs.databricks.com/machine-learning/model-inference/serverless/serverless-real-time-inference.html) in Databricks.
+
+## Example Usage
+
+```hcl
+resource "databricks_serving_endpoint" "this" {
+  name = "ads-serving-endpoint"
+  config {
+    served_models {
+      name = "prod_model"
+      model_name = "ads-model"
+      model_version = "2"
+      workload_size = "Small"
+      scale_to_zero_enabled = true
+    }
+    served_models {
+      name = "candidate_model"
+      model_name = "ads-model"
+      model_version = "4"
+      workload_size = "Small"
+      scale_to_zero_enabled = false
+    }
+    traffic_config {
+      routes {
+        served_model_name = "prod_model"
+        traffic_percentage = 90
+      }
+      routes {
+        served_model_name = "candidate_model"
+        traffic_percentage = 10
+      }
+    }
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the serving endpoint. This field is required and must be unique across a workspace. An endpoint name can consist of alphanumeric characters, dashes, and underscores.
+* `config` - (Required) The serving endpoint configuration.
+
+### config Configuration Block
+
+* `served_models` - (Required) Each block represents a served model for the endpoint to serve. A serving endpoint can have up to 10 served models.
+* `traffic_config` - The traffic split configuration amongst the served models.
+
+### served_models Configuration Block
+
+* `name` - The name of a served model. It must be unique across an endpoint. If not specified, this field will default to -. A served model name can consist of alphanumeric characters, dashes, and underscores.
```

Comment:
I thought it defaults to modelname-modelversion?

Reply:
Pulled the field description from the REST API docs here: https://docs.databricks.com/dev-tools/api/latest/endpoints.html#operation/create